src/tic-tac-toe.ts

```diff
@@ -0,0 +1,24 @@
+type Square = string;
+type Row = Square[];
+type Board = Row[];
+
+class Game {
+  public board: Board;
+  private currentPlayer: string;
+  constructor() {
+    this.currentPlayer = "X";
+    this.board = [["", "", ""], ["", "", ""], ["", "", ""]];
+  }
+
+  switchPlayers() {
+    if (this.currentPlayer === "X") this.currentPlayer = "O";
+    else this.currentPlayer = "X";
+  }
+
+  place(x: number, y: number) {
+    this.board[x][y] = this.currentPlayer;
```

Comment:
I like that this implementation could potentially allow for more than two players!